convert2rhel/actions/pre_ponr_changes/backup_system.py

```diff
@@ -77,8 +80,57 @@ def run(self):
 
         super(BackupRepository, self).run()
 
-        repo.backup_yum_repos()
-        repo.backup_varsdir()
+        self.backup_yum_repos()
+
+    def backup_yum_repos(self):
+        """Backup .repo files in /etc/yum.repos.d/ so the repositories can be restored on rollback."""
+        loggerinst.info("Backing up .repo files from %s." % DEFAULT_YUM_REPOFILE_DIR)
+
+        repo_files_backed_up = False
+
+        for repo in os.listdir(DEFAULT_YUM_REPOFILE_DIR):
+            if repo.endswith(".repo") and repo != "redhat.repo":
+                repo_path = os.path.join(DEFAULT_YUM_REPOFILE_DIR, repo)
+                restorable_file = RestorableFile(repo_path)
+                backup.backup_control.push(restorable_file)
+                repo_files_backed_up = True
+
+        if not repo_files_backed_up:
+            loggerinst.info("No .repo files backed up.")
+
+
+class BackupVariables(actions.Action):
+    id = "BACKUP_VARIABLES"
```

Comment:
I would change this name a bit, maybe `BackupPackageManagerVariables` (?)

Just to make it more explicit to tell what we are backing up, variables could be anything, anywhere.

Reply:
Could also be `BackupYumVariables` as well... I don't think it does matter in that context if we say yum/dnf variables, as they are treat as one.